src/Repository.js

```diff
@@ -55,6 +55,9 @@ export default class Repository {
   }
 
   get packageConfigs() {
+    if (this.lernaJson.useWorkspaces) {
+      return this.packageJson.workspaces;
```

Comment:
` || [DEFAULT_PACKAGE_GLOB];`? This is not mentioned in the readme, so everything went boom when I tried this